lib/apipie/error_description.rb

```diff
@@ -31,10 +31,10 @@ def initialize(code_or_options, desc=nil, options={})
       end
     end
 
-    def to_json
+    def to_json(lang)
```

Comment:
maybe the param should have a default? I wonder if this cannot break people's applications...